content/zh/examples/admin/dns/busybox.yaml

```diff
@@ -1,14 +1,14 @@
-apiVersion: v1
-kind: Pod
-metadata:
-  name: busybox
-  namespace: default
-spec:
-  containers:
-  - name: busybox
-    image: busybox:1.28
-    command:
-      - sleep
-      - "3600"
-    imagePullPolicy: IfNotPresent
-  restartPolicy: Always
+apiVersion: v1

```

Comment:
can you articulate what have been changed?

Reply:
Sorry, wrong. revert it

Reply:
Problems with delimiters in windows and unix